led_strip/CMakeLists.txt

```diff
@@ -1,22 +1,32 @@
 include($ENV{IDF_PATH}/tools/cmake/version.cmake)
 
 set(srcs "src/led_strip_api.c")
+set(public_requires)
 
+# Starting from esp-idf v5.x, the RMT driver is rewritten
 if("${IDF_VERSION_MAJOR}.${IDF_VERSION_MINOR}" VERSION_GREATER_EQUAL "5.0")
     if(CONFIG_SOC_RMT_SUPPORTED)
         list(APPEND srcs "src/led_strip_rmt_dev.c" "src/led_strip_rmt_encoder.c")
     endif()
-else() 
+else()
+    message(WARNING "esp-idf v4.x has reached end-of-life. The compatibility with v4.x will be removed in the next major release.")
```

Comment:
I think the warning is probably unnecessary: if the future release removes v4.4 compatibility, the component manager should still be able to install the last compatible release.